company_website/view_helpers.py

```diff
@@ -24,3 +28,15 @@ def add_meta_tags_to_page_context(page_name: str, context_data: Dict) -> Dict:
     context_data["meta_description"] = page_seo.meta_description
     context_data["keywords"] = page_seo.keywords
     return context_data
+
+
+def generate_subresource_integrity_sha384(content_link: str) -> str:
```

Comment:
You can change `content_link` just to `url` because finally it is always url :)

Reply:
Done. "Content..." seams more verbose, but whatever :P